src/pages/Markets/components/AssetGridWithData.tsx

```diff
@@ -31,7 +35,8 @@ export const AssetGridWithData = ({
   } = dataQuery({
     enabled: inView,
     // ts isn't smart enough to narrow this down, and we don't want to pass it for all categories where this does not apply
```

Comment:
I think this comment is now irrelevant with the below change.

Reply:
Ah yeah nice catch! Removed in [1ac600a](https://github.com/shapeshift/web/pull/8006/commits/1ac600aeb95da5729ef78865c837fd0bcec0f777)